src/overview/main.d

```diff
@@ -1,8 +1,91 @@
+/*******************************************************************************
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
 module overview.main;
 
 import octod.core;
 import octod.api.repos;
+import overview.repository;
 
 void main ( )
 {
+    import overview.htmlreport;
+    import semver.Version;
+    import vibe.core.log : logInfo;
+    import std.algorithm.iteration : map, filter, each;
+    import std.range : array;
+
+    Configuration conf;
+    conf.dryRun = false;
+    conf.oauthToken = "READ-ME-FROM-CONFIG";
```

Comment:
I don't know exactly how this is used but it would be nice for this tool to be able to run without authentication (you can still access public repos as a unauthenticated user).